### DISCUSSION TITLE
you can chart my peakophobia song Jordan

### DISCUSSION THREAD

**Discussion by daveandbambifan1234** [JordanSantiagoYT/FNF-JS-Engine#677] [General]
My Discord is daveandbambifan_72402 so I can give you the character (:
info:
remember this is an idea, also the song isn't spammy on the player side so it will be playable for the player, don't know why I made this song and also this song is not songs like "old/V1 annihilation" where it's very high pitch on  vocals, and it also doesn't break the sound barrier, I worked pretty hard on this song so please chart it with your hearts content
https://www.youtube.com/watch?v=tEaGbcPg2UI (the song, 130 or 260 BPM)
if you also feel like making a cover I can give you the flp, just let me know if you're missing some files, i will give you some important ones in the Google drive thing, if you don't have any tiny details in the flp I will give you those small details

**Comment by daveandbambifan1234** [comment]
⚠warning⚠: I am here to let you know that you can chart my song for a video idea, this is not something else and I just gave you an idea on something to chart if you're bored or something, but if you do end up charting this make sure to credit me because I worked pretty hard on this song, have a nice day for whoever is reading this or looking at this info

**Comment by NAEL2XD** [comment]
this is not a way to request songs.

**Comment by TheStinkern** [comment]
request it through here:

https://forms.gle/X5246W1iVZDpE5cc6
